README.md

```diff
@@ -77,7 +77,7 @@ through the `OperatorHub` section of the OpenShift web console. To deploy with O
 > You can combine all or multiple `make` commands in one single command if you don't have anything to modify in between the steps.
 >
 > ```sh
-> make docker-build docker-push bundle bundle-build bundle-push catalog-fbc-build catalog-push catalog-deploy IMAGE_TAG_BASE=quay.io/musername/rh-openshift-builds/operator
+> make docker-build docker-push bundle bundle-build bundle-push catalog-fbc-build catalog-build catalog-push catalog-deploy IMAGE_TAG_BASE=quay.io/musername/rh-openshift-builds/operator
```

Comment:
Item for future improvement - perhaps a `deploy-all` target that runs all these steps in order?